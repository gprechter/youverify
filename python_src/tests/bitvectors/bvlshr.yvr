a,b,c,d,e,f,g: BV[4]
a = BV{2, 4}
b = BV{4, 4}
call c = bvlshr(b, a)
call d = bvlshr(BV{15, 4}, a)
call e = bvlshr(BV{15, 4}, b)
call f = bvlshr(BV{8, 4}, a)
call g = bvlshr(BV{8, 4}, BV{3, 4})