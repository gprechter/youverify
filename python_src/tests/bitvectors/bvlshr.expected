{
    "a": 2,
    "b": 4,
    "c": 1,
    "d": 3,
    "e": 0,
    "f": 2,
    "g": 1
}